### ISSUE TITLE
Empty black screen appear when screen-share on second time

### ISSUE THREAD

**Issue by ashiqhassan95** [jitsi/jitsi-meet#11589]
### Description:

An empty black screen appears for firefox users when the presenter shares the desktop screen for the second time and consecutive.

### Steps to reproduce:

1. User A from the chrome browser create and join a meeting room.
2. User B joins the same meeting from the Firefox browser.
3. User A starts screen share and User B can see the shared screen.
4. User A stops screen share and User B cannot see it (that's okay).
5. Now User A again starts screen share.
6. User B is not able to see the shared screen and User B just sees an only empty black screen.
7. User B has rejoin in order to see the shared video.

### Expected behavior:

User B should be able to view video shared by User A.

### Actual behavior:

User B should not able to view video shared by User A on second time.

![image](https://user-images.githubusercontent.com/2870845/170682355-b580dd99-cc5d-417b-b0be-c0eec915e862.png)

![image](https://user-images.githubusercontent.com/2870845/170682382-6c23de2a-9a2c-4850-b614-5a4a81873218.png)

### Server information:

- Jitsi Meet version: [meet.jit](https://meet.jit.si/)
- Operating System:

### Client information:

- Browser / app version: Chrome Version 101.0.4951.64 | Firefox 100.0 (64-bit)
- Operating System: Ubuntu

**Comment by saghul** [comment]
I could not reproduce this with Firefox 101 beta.

**Comment by ashiqhassan95** [comment]
> I could not reproduce this with Firefox 101 beta.

Thank you for the reply.

I will check same from my side too.

**Comment by utopos** [comment]
I confirm. on Firefox 101.0.1 (64-bit) - Arc Linux. Need to reconnect each time to see the share.

**Comment by Sudland** [comment]
Had the same problem, but is gone now on the newest release of Jitsi.

Firefox 101.0.1 Gentoo

**Comment by ppenguin** [comment]
Necro-bumping, because this is probably a server side config that was solved at the public jitsi site, but I'm experiencing it on a self-hosted instance.

So hoping that someone knowledgeable about jitsi config has an idea?

**Comment by saghul** [comment]
Not necessarily. Are you running the latest stable release?

**Comment by ppenguin** [comment]
"Not really", i.e. I used the nixos module on a fresh nixos server, and then went down the rabbit hole of fixing authentication and lobby etc. (instead of "giving up" sooner and just using docker). Version corresponds to `jitsi-meet-web-1.0.6644`, I assume the other componets are from the same date. 

Since I'm "almost there", I figured I might as well use the result to contribute improvements to the nixos module, but it appears more tricky than expected...
